pandas/core/strings/base.py

```diff
@@ -246,7 +246,9 @@ def _str_removesuffix(self, suffix: str) -> Series:
         pass
 
     @abc.abstractmethod
-    def _str_split(self, pat=None, n=-1, expand: bool = False):
+    def _str_split(
+        self, pat=None, n=-1, expand: bool = False, regex: bool | None = None
```

Comment:
is it documented somewhere what regex=None means?

Reply:
Yup, this is in the public docstring https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.str.split.html